chart/jenkins-operator/values.yaml

```diff
@@ -1,32 +1,144 @@
-# Default values for jenkins-operator.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
+# Jenkins Custom Resource (CR) Helm chart
 
+# jenkins configures pod instance
 jenkins:
+  # enabled can enable or disable the jenkins instance
+  # Set to false if you have configured CR already and/or you want to deploy only operator
   enabled: true
 
+  # apiVersion is version of the CR manifest template
+  # The recommended and default value is "jenkins.io/v1alpha2"
```

Comment:
please add a link to the documentation with available api versions

Reply:
```suggestion
  # link to the scheme
```